singer-io/tap-activecampaign
# Fix pylint issues

# Description of change
Fix pylint issues. Incorporate circleci and tests PRs.

# Manual QA steps
Ran pylint and fixed issues. Ran singer-discover, singer-check-tap, target-stitch (initial and incremental load) for all endpoints. No issues.
 
# Risks
Low. Tap currently in Alpha.
 
# Rollback steps
Revert to v0.0.1
